addons/point_of_sale/static/src/app/navbar/navbar.xml

```diff
@@ -53,6 +53,12 @@
                         <DropdownItem t-if="this.env.debug" onSelected="() => debug.toggleWidget()">
                             Debug Window
                         </DropdownItem>
+                        <DropdownItem t-if="this.env.debug" onSelected="() => pos.toggleProductCoverMode()">
+                            <t t-esc="pos.productCoverMode ? 'Fit Image Inside' : 'Fill Image to Edges'"/>
```

Comment:
```suggestion
                            <t t-if="pos.productCoverMode">Fit Image Inside</t>
                             <t t-else="">Fill Image to Edges</t>
                                 
```

Put the text as a child of an element so that it gets translated.